code/modules/ghostroles/spawner/base.dm

```diff
@@ -93,13 +105,11 @@
 		enabled = FALSE
 	return TRUE
 
-//This proc selects the spawnpoint to use.
-/datum/ghostspawner/proc/select_spawnpoint(var/use=TRUE)
-	if(length(spawn_atoms))
-		var/chosen_spawn_atom = pick(spawn_atoms)
-		var/turf/T = get_turf(chosen_spawn_atom)
-		if(T)
-			return T
+//This proc selects the spawnpoint to use. - Only used when mode is GS_LOC_POS
+/datum/ghostspawner/proc/select_spawnlocation(var/use=TRUE)
+	if(loc_type != GS_LOC_POS)
+		log_debug("Ghostspawner: select_spawnlocation is not valid for spawner [short_name] as it is not position pased")
+		return null
```

Comment:
```suggestion
		return
```

Reply:
null is quite intentional here, as I dont want to return . in case. is used some time later on.